crates/hc_sandbox/src/save.rs

```diff
@@ -78,7 +81,7 @@ pub fn clean(mut hc_dir: PathBuf, sandboxes: Vec<usize>) -> anyhow::Result<()> {
 }
 
 /// Load sandbox paths from the `.hc` file.
-pub fn load(mut hc_dir: PathBuf) -> anyhow::Result<Vec<PathBuf>> {
+pub fn load(mut hc_dir: PathBuf) -> std::io::Result<Vec<Option<PathBuf>>> {
```

Comment:
Prefer using std::io::Result, since IO errors is all this code can return anyway. Using `anyhow` when consuming the sandbox as a library is difficult. Prefer only using anyhow in binary-only code.